packages/react/src/constants/device-sizes.constant.ts

```diff
@@ -36,3 +36,21 @@ export const sizes = {
     return 'large';
   },
 };
+
+export const useSizes = ({ small, medium }: { small?: number; medium?: number }) => {
```

Comment:
I think a name like `setBreakpoints` is a little more descriptive of what is happening
```suggestion
export const setBreakpoints = ({ small, medium }: { small?: number; medium?: number }) => {
```

Reply:
just removed this one and created another one for diff purpose